Networking/Networking/Remote/WooShippingRemote.swift

```diff
@@ -114,7 +121,50 @@ public final class WooShippingRemote: Remote, WooShippingRemoteProtocol {
                                          availableAsRESTRequest: true)
 
             let mapper = WooShippingPackagesMapper()
+            enqueue(request, mapper: mapper, completion: completion)
+        } catch {
+            completion(.failure(error))
+        }
+    }
 
+    /// Initiates a shipping label purchase.
+    ///
+    /// This request returns the label purchase data, including a `PURCHASE_IN_PROGRESS` status.
+    /// After initiating the purchase, we must poll the backend for the updated label status (successful purchase or error).
```

Comment:
The new endpoint returns significantly more detail than the legacy endpoint in the response, but we still need to use this approach to poll for the completed purchase. The data in `ShippingLabelPurchase` is sufficient for that — we don't need to decode the additional data.